### PR TITLE
Link Hiba.cpp

### PR DESCRIPTION
**Pull request by HeartPiece44**
Matches and links Hiba.cpp and fixes related headers.

### CODE REVIEW

File: include/efx/THibaFire.h [hunk 1]
```diff
@@ -5,6 +5,11 @@
 
 namespace efx {
 struct THibaFire : public TForever4 {
```
Comment (EpochFlame) [inline include/efx/THibaFire.h:7]: use enums for particle id here, please

**Review by EpochFlame — CHANGES_REQUESTED** [review]
looks good other than the header needing to use particle id enum- see other comments

File: include/efx/THibaFire.h (hunk 1 above)
Comment (HeartPiece44) [reply include/efx/THibaFire.h:7]: have revamped particle ID system of repo c:

**Review by EpochFlame — APPROVED** [review]
lgtm

**Pull request merged**